spec/factories.rb

```diff
@@ -317,6 +344,11 @@
     explanation 'Some explanation'
   end
 
+  factory :auto_grading_keyword_option, class: Assessment::AutoGradingKeywordOption do
+    keyword 'keyword1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.